modules/upgrade/pages/proxy-upgrade.adoc

```diff
@@ -0,0 +1,50 @@
+[[update.suse.manager.proxy]]
+= Update {productname} Proxy
+
+{productname} Proxies are managed in the same way as clients.
+Maintenance updates (MU) can be installed on a {productname} Proxy in the same way as other clients.
+Some updates will require a restart of the proxy service.
+
+// a remote command (or a salt state) can perform the restart.
+// Creating an Action Chain can help with this task.
+// With an action chain you can update proxies on batches, or all at the same time:
+// These are the basic steps to run:
+////
+1. Run a salt command: `spacewalk-proxy stop`
+2. Update the packages
+3. Run a salt command: `spacewalk-proxy start`
+////
+
+
+
+== Prepare to Update
+
+Before you perform any proxy upgrade, schedule a maintenance window.
+The clients registered to {productname} through the proxy will not be able to connect to {productname} while the upgrade is in progress.
+For more information about maintenance windows, see xref:administration/maintenance-window.adoc[].
+
+
+
+
+== Update the Proxy
+
+To update a proxy you first stop the proxy service, then update the software and finally restart the proxy service.
+
+.Procedure: Updating the {productname} Proxy
+
+. On the {productname} Server where the proxy is registered as a client, stop the proxy service:
++
+----
+spacewalk-proxy stop
+----
+
+. Select the packages to be updated on the proxy.
```

Comment:
Shouln't we just say "Update the packages on the proxy" to be more generic?

Otherwise the user should "select the packages", and then "apply the selection".

Reply:
I vote for the latter. In a procedure, it's all about the verbs :wink:

Reply:
done